src/manifolds/Rotations.jl

```diff
@@ -651,7 +654,6 @@ function retract!(M::Rotations, q::AbstractArray{T}, p, X, method::QRRetraction)
     D = Diagonal(sign.(d .+ convert(T, 0.5)))
     return copyto!(q, qr_decomp.Q * D)
 end
-retract!(M::Rotations, q, p, X) = retract!(M, q, p, X, QRRetraction())
```

Comment:
I don't know. I don't recall deleting it on purpose

Reply:
I _did_ remove this default on purpose. It interferes too much with group manifolds and gives a strange behaviour concerning the default retraction. Even sending both (mutating and nonmutating first to `:parent:` did not help; without setting this as the default retraction (i.e. keeping exp) all tests work. This might not be the best option and we should investigate; for now the default is back to `ExponentialRetraction`.

Reply:
and in this form it was even inconsistent in the sense that the `inverse_retraction` did not have the corresponding inverse as default.